PhpAmqpLib/Message/AMQPMessage.php

```diff
@@ -56,5 +53,7 @@ public function __construct($body = '', $properties = null)
     public function setBody($body)
     {
         $this->body = $body;
+
+        return $this;
```

Comment:
@videlalvaro What parent class?  `GenericContent` doesn't seem to have `setBody`